timemachine/fe/rbfe.py

```diff
@@ -617,47 +602,43 @@ def estimate_relative_free_energy_via_greedy_bisection(
 
 
 def run_vacuum(
-    mol_a,
-    mol_b,
-    core,
-    forcefield,
+    mol_a: Chem.rdchem.Mol,
+    mol_b: Chem.rdchem.Mol,
+    core: NDArray,
+    forcefield: Forcefield,
     _,
-    n_frames,
-    seed,
-    n_eq_steps=10000,
-    steps_per_frame=400,
-    n_windows=None,
-    min_cutoff=None,
+    seed: int,
+    md_params: MDParams = DEFAULT_MD_PARAMS,
+    n_windows: Optional[int] = None,
+    min_cutoff: Optional[float] = None,
 ):
+    assert md_params is not None and md_params.local_steps == 0, "vacuum simulations don't support local steps"
```

Comment:
Not sure about this. It means the caller will need to construct a different md_params for each leg.

Reply:
Originally was a warning that changed the local steps to 0, changed here [98a2c1d](https://github.com/proteneer/timemachine/pull/1026/commits/98a2c1d97b61405788fcee301953f778c4736450)

No strong preference, easily handled in production

Reply:
Reverted and went back to the warning. Easy to miss, but for vacuum we rarely care

Reply:
Q: does this mean there will be a spurious warning under normal operation when running RBFE?

Reply:
If we use the same MDParams for all three stages and they request local moves then yes. We can avoid this if we want when we run, doesn't seem to make much difference. :shrug: